pages/common/aws-s3api.md

```diff
@@ -0,0 +1,36 @@
+# aws s3api
+
+> Create and delete s3 buckets, and edit bucket properties.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/s3api/>.
```

Comment:
the link says `latest` but it's a liar — the page itself says that it's for v1. Use this link instead:
```suggestion
> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3api/index.html>.
```